### ISSUE TITLE
[4.3.1]: email template + pdf not being sent if field fileNameFormat is empty

### ISSUE THREAD

**Issue by jarheadcore** [craftcms/commerce#3309]
### Description
If a commerce order has a email template + order pdf attached, the empty field "Order PDF Filename Format" / field id: fileNameFormat generates the following error when trying to send the order mails:

`craft\web\View::renderObjectTemplate(): Argument #1 ($template) must be of type string, null given, called in /app/vendor/craftcms/commerce/src/services/Emails.php on line 694`

### Steps to reproduce
1. place an order
2. use html email template
3. attach order pdf to it
4. set pdf name, handle + path but leave fileNameFormat field empty

### Expected behavior
As in craftcms/commerce/src/services/email.php line 696 -> `$fileName = $pdf->handle . '-' . $order->number;` fallback if !filename, the pdf's handle + ordernumber should be set as filename.

### Actual behavior
the queue job runs into a timeout and emails (+ pdf attachments) are not sent.

code: https://share.cleanshot.com/79TCrNFL
actual issue / timeout queue: https://share.cleanshot.com/KyfxJCf2
queue error: https://share.cleanshot.com/rq0G7KDv

Not having the issue if providing a nameformat for the pdf.
Would it be possible to make it a required field?

### Craft CMS version

4.5.7

### Craft Commerce version

4.3.1

### PHP version

8.2

### Operating system and version

bento/debian-11

### Database type and version

mysql 8.0

**Comment by lukeholder** [comment]
Thanks for reporting. This has been fixed for the next release.

To get the fix early, change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "dev-develop#9748053478f4bc32905c1536e73ce112def7a91e as 4.3.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by lukeholder** [comment]
This is now fixed in 4.3.2
